_translator-files/po/de/Tips-Tricks-More.po

```diff
@@ -45,116 +47,118 @@ msgid ""
-msgstr ""
+msgstr "Mehrere Benutzer haben berichtet, dass es ihnen gelungen ist, ein \"virtuelles Publikum\" für eine Jamulus-Sitzung zu schaffen, indem sie [JACK Audio](https://jackaudio.org) verwendet haben, um das Jamulus-Signal über JackRouter an die Zielanwendung (in diesem Fall Zoom-Meetings) zu leiten."
 
 #. type: Plain text
 #: ../wiki/en/Tips-Tricks-More.md:35
 msgid "You can also use [VoiceMeeter](https://www.vb-audio.com/Voicemeeter/banana.htm) (Banana) for Windows or [BlackHole](https://github.com/ExistentialAudio/BlackHole) for macOS to route the Jamulus output to multiple destinations, for example to your headphones and the meeting application at the same time."
-msgstr ""
+msgstr "Du kannst auch [VoiceMeeter](https://www.vb-audio.com/Voicemeeter/banana.htm) (Banana) für Windows oder [BlackHole](https://github.com/ExistentialAudio/BlackHole) für macOS verwenden, um die Jamulus-Ausgabe an mehrere Ziele zu leiten, z. B. an deine Kopfhörer und die Meeting-Anwendung gleichzeitig."
 
 #. type: Plain text
 #: ../wiki/en/Tips-Tricks-More.md:37
 msgid "### Recording Jamulus on Windows with Reaper"
-msgstr ""
+msgstr "### Aufnahme von Jamulus unter Windows mit Reaper"
 
 #. type: Plain text
 #: ../wiki/en/Tips-Tricks-More.md:39
 msgid "Jamulus user [Rob Durkin](https://sourceforge.net/u/bentwrench/profile/) has written a [guide to recording the output of Jamulus](https://docs.google.com/document/d/1tENfNKTWHasuTg33OdLLEo4-OOhWJolOo42ffSARxhY/edit) (Google Doc) using the ReaRoute add-on for [Reaper](https://www.reaper.fm/)."
-msgstr ""
+msgstr "Jamulus-Benutzer [Rob Durkin](https://sourceforge.net/u/bentwrench/profile/) hat eine [Anleitung zur Aufzeichnung der Jamulus-Ausgabe](https://docs.google.com/document/d/1tENfNKTWHasuTg33OdLLEo4-OOhWJolOo42ffSARxhY/edit) (Google Doc) unter Verwendung des ReaRoute-Add-ons für [Reaper](https://www.reaper.fm/) geschrieben."
 
 #. type: Plain text
 #: ../wiki/en/Tips-Tricks-More.md:41
 msgid "### Sharing song/chord sheets"
-msgstr ""
+msgstr "### Gemeinsame Nutzung von Lied-/Akkordblättern"
 
 #. type: Plain text
 #: ../wiki/en/Tips-Tricks-More.md:43
 msgid "Jamulus user [BTDT](https://sourceforge.net/u/btdt/profile/) has written a system called [305keepers](https://github.com/keepers305/Song-Sheet-Sharing-Web-Pages), a web application that allows a \"Jam leader\" to push song sheets (in PDF format) to \"Jammers\" in real time using standard web browsers."
-msgstr ""
+msgstr "Jamulus-Benutzer [BTDT](https://sourceforge.net/u/btdt/profile/) hat ein System namens [305keepers](https://github.com/keepers305/Song-Sheet-Sharing-Web-Pages) geschrieben, eine Webanwendung, mit der ein \"Jam-Leader\" Songblätter (im PDF-Format) in Echtzeit über Standard-Webbrowser an \"Jammer\" senden kann."
```

Comment:
Should go to the Jamulus related projects file in the main repo...

https://github.com/jamulussoftware/jamulus/blob/master/docs/RELATED-PROJECTS.md

@gilgongo

Reply:
I can't remember why Related Projects isn't in Tips & Tricks. They seem to overlap.

Reply:
Agree. We should probably move the stuff from Tips and Tricks to the related projects file. Can you open an issue?